workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Commands/Deployment.cs

```diff
@@ -79,7 +79,7 @@ public static class Deployment
 
             return new WrappedTask<Result<List<DeploymentInfo>, Ipc.Error>, string>
             {
-                Task = RunDeploymentLauncher(args, OutputRedirectBehaviour.RedirectStdErr, token,
```

Comment:
do we not want to output it to stderr?

Reply:
It still outputs to stderr, but we use it for the inter process communication so we don't want it auto redirected into the Unity console